### DISCUSSION TITLE
DALL-E via Pipeline sample

### DISCUSSION THREAD

**Discussion by reiebrole30** [open-webui/open-webui#2907] [General]
![image](https://github.com/open-webui/open-webui/assets/139945395/4f0cdcbc-ff89-4175-9e15-b10e508f779e)

with the DALL-E via pipelines, who needs a monthly sub now when you can just use a pay as you go via API, Thank you so much for all your hardwork

**Comment by Hugobox** [comment]
Cool! Is this pipeline shared somewhere? If not care to share it?
